Lab2_Code.m

```diff
@@ -0,0 +1,204 @@
+rconv = multiconv(rbg,20);
+rx = 0:length(rconv)-1;
+plot(rx/20,rconv); hold on;
+
+% Average over 30 observing intervals
+rconv = multiconv(rbg,30);
+rx = 0:length(rconv)-1;
+plot(rx/30,rconv); hold on;
+
+% Average over 40 observing intervals
+rconv = multiconv(rbg,40);
+rx = 0:length(rconv)-1;
+plot(rx/40,rconv); hold on;
+
+title("Average Probability Distribution After N Intervals"); 
+xlabel("Observations"); ylabel("Probability"); 
+legend("N = 10","N = 20","N = 30","N = 40"); hold off;
+
+%% Section 2B
+% According to the Central Limit Theorem, the distribution should
+% approximate a normal distribution when the sample size gets large. As the
+% average over more observing intervals increased, the Rayleigh
+% distribution changed. Specifically, the width of the distribution became
+% thinner, the amplitude at the mean decreased, and the shape became
+% smoother. However, the mean did not shift. The Rayleigh distribution
+% approached a Gaussian distribution after approximately 30 intervals. 
+
+%% Problem 3
+% For this section, I selected the width of the background Gaussian
+% distribution as X = 5 and the signal of strength as Y = 8.6. 
+
+%% Section 3A (Version 1)
+% Here, we make a Gaussian distribution with mu = 0 and sigma = 5.
+% Afterwards, we calculate the probability of an 8.6 signal using the cdf()
+% function. We put this value into the norminv() function calculate the
+% sigma. From this, we get a sigma of 1.72 which means we cannot claim a
+% discovery since that's associated with 5-sigma. 
+gdist = makedist("Normal",'mu',0,'sigma',5);
+gprob = cdf(gdist,8.6);
+gsigma = norminv(gprob);
+disp(gsigma);
+
+%% Section 3B (Version 2)
+% The statistical question I want to ask is...
+gdist = makedist("Normal",'mu',0,'sigma',5);
+gpdf = pdf(gdist,-30:30);
+gprobf = gpdf*10000;
+gbg = plot(-30:30,gprobf);
+title("Probability Distribution of Background");
+xlabel("Observations"); ylabel("Probability");
+
+% The probability we get at least one signal-like event from the background
+% is the probability we get from one of these pixels. If we assume the
+% distributions for each pixel are identical and independent from one
+% another, we simply multiply the pdf of one pixel by 10k. 
+
+%% Section 3C (Version 2) % iNCOrrECT
+% Calculates the signifance of the detection of the brightest candidtate 
+% pixel from the region with signal 8.6. 
+gdist = makedist("Normal",'mu',0,'sigma',gsigma);
+gprob = cdf(gdist,8.6);
+gsigma = norminv(gprob);
+disp(gsigma);
+
+%% Problem 4
+
+%% Section 4A
+% Calculates the signal required for a 5-sigma detection in Problem 3A
+% Version 1. Uses norminv() function with mu=0 and sigma=5. 
+gsignal = norminv(normcdf(5),0,5);
+disp(gsignal);
+
+%% Section 4B %INCORRECT
+% Calculates the signal required for a 5-sigma detection in Problem 3C
+% Version 2. Uses norminv() function with mu=0 and sigma=5. 
+gsignal = norminv(normcdf(5),0,500);
+disp(gsignal);
+
+%% Section 4C
+% 
+
+%% Section 4D
+% 
```

Comment:
Let me know when you push the rest and I will go over it as soon as I can! This is all solid work! and if I'm being to Nit-picky feel free to tell me to chill!